config/locales/en.yml

```diff
@@ -95,12 +95,21 @@ en:
     logout: logout
     permissions: "Permissions"
     permissions_heading: "Your Permissions"
+    change_password_heading: "Change Your Password"
     access: "Access"
     client_switch_heading: "Switch Authentication Clients"
     client_switch_legend: "In development you can switch between using cognito and the stub authentication clients. If you use the stub client you can assume different roles."
     banner: "There are features on this page which are only avialable in the development environment."
     assume_roles_heading: "Switch Roles"
     assume_roles_legend: "Select which role(s) you wish to assume?"
+    old_password_mismatch: "Your old password does not match the one we have on record"
+    new_password_mismatch: "Your new password did not match your confirmation password"
+    password_changed: "Password changed successfully"
+    change_password_legend: "You can change your password by entering your current password followed by a new password in the space provide and the same new password again in the confirmation space."
+    current_password_lbl: "Current password:"
```

Comment:
all of this probably should be it's own section within the profile.. it's getting too big